src/containers/templates/SignUp/SignUp.js

```diff
@@ -1,20 +1,26 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import SignUp from 'components/templates/SignUp';
 
-class SignUpContainer extends Component {
-  render() {
-    return (
-      <SignUp
-        listeners={{
-          onClickFacebook: () => console.log('fb click'),
-          onSubmitSignUp: e => {
-            e.preventDefault();
-            console.log('signup click');
-          }
-        }}
-      />
-    );
-  }
-}
+const SignUpContainer = ({ history }) => {
+  return (
+    <SignUp
+      listeners={{
+        onClickFacebook: () => console.log('fb click'),
+        onSubmitSignUp: e => {
+          e.preventDefault();
+          console.log('signup click');
+          history.push('/experiences');
```

Comment:
@AlexGilleran , if you want to add the `/experiences` to facebook button just do it in above one in `onClickFacebook`